services/order.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Order, User, MovieSession, Ticket
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict], username: str, date: datetime = None
```

Comment:
The type hint `list[dict]` is used here. This syntax is only compatible with Python 3.9 and above. For broader compatibility, consider using `List[Dict]` from the `typing` module.